api/src/main/java/org/openmrs/module/sync2/client/rest/resource/Person.java

```diff
@@ -170,4 +171,57 @@ public Date getBirthtime() {
     public void setBirthtime(Date birthtime) {
         this.birthtime = birthtime;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        Person person = (Person) o;
+        return Objects.equals(uuid, person.uuid) &&
+                Objects.equals(display, person.display) &&
+                Objects.equals(preferredName, person.preferredName) &&
+                Objects.equals(preferredAddress, person.preferredAddress) &&
+                Objects.equals(names, person.names) &&
+                Objects.equals(addresses, person.addresses) &&
+                Objects.equals(birthdateEstimated, person.birthdateEstimated) &&
+                Objects.equals(age, person.age) &&
+                Objects.equals(gender, person.gender) &&
+                Objects.equals(dead, person.dead) &&
+                Objects.equals(causeOfDeath, person.causeOfDeath) &&
+                Objects.equals(deathDate, person.deathDate) &&
+                Objects.equals(voided, person.voided) &&
+                Objects.equals(deathdateEstimated, person.deathdateEstimated);
+    }
```

Comment:
I think that you should create a ticket and comment why you don't compare birthdate and birthtime

Reply:
Yes, I'll do that.